Frontend/ui-driver/src/Flow.purs

```diff
@@ -851,7 +851,7 @@ driverProfileFlow = do
     GO_TO_CALL_DRIVER state -> do
       modifyScreenState $ DriverProfileScreenStateType (\driverProfileScreen -> state {props = driverProfileScreen.props { alreadyActive = false}})
       _ <- Remote.callDriverToDriverBT  state.data.rcNumber
-      pure $ toast $ (getString PLEASE_WAIT_WHILE_WE_CONNECT_WITH_DRIVER)
```

Comment:
if not required remove `PLEASE_WAIT_WHILE_WE_CONNECT_WITH_DRIVER` from translations also